### ISSUE TITLE
Cluster verify behaviour in face of node errors

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#146]
Originally reported of Google Code with ID 90.

```
LUVerifyCluster's Exec() function will exit early the node verification if 
some of the requested keys are in error. For example, failure to verify the 
hypervisor will skip the time checks.

The code should deal better with incomplete responses, and not skip checks 
for which it has data.
```
Originally added on 2010-03-12 08:39:50 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2010-03-12 22:07:50 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Defect

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2010-03-16 10:16:04 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been implemented in 2.1.4 or so.

Marking as closed.

iustin
```
Originally added on 2010-07-28 23:16:11 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
